subsys/bluetooth/controller/hci_internal.h

```diff
@@ -24,6 +27,45 @@
  */
 int hci_internal_cmd_put(uint8_t *cmd_in);
 
+/** A user implementable HCI command handler
+ *
+ * What is done in the command handler is up to the user.
+ * Parameters can be returned to the host through the raw_event_out output parameter.
+ *
+ * When the command handler returns, a Command Complete or Command Status event is generated.
+ *
+ * @param[in]  cmd               The HCI command itself. The first byte in the buffer corresponds
+ *                               to OpCode, as specified by the Bluetooth Core Specification.
+ * @param[out] raw_event_out     Parameters to be returned from the event as return parameters in
+ *                               the Command Complete event.
+ *                               Parameters can only be returned if the generated event is
```

Comment:
should be rephrased now

Reply:
Why?

Reply:
cannot it return Command Status as well?

Reply:
The Command Status event does not include return parameters